Makefile

```diff
@@ -144,21 +149,16 @@ controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessar
 $(CONTROLLER_GEN): $(LOCALBIN)
 	test -s $(LOCALBIN)/controller-gen || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
 
+.PHONY: lint
+lint: golangci-lint
+	 $(GOLANGCI_LINT) run --issues-exit-code $(GOLANGCI_EXIT_CODE)
```

Comment:
Nice! This allows us to run the linter in "warn" mode until we resolve all lint issues. 🙂